packages/eventsub-http/src/EventSubHttpBase.ts

```diff
@@ -258,8 +258,6 @@ export abstract class EventSubHttpBase extends EventSubBase {
 				res.setHeader('Content-Type', 'text/plain');
 				res.writeHead(410);
 				res.end('Not OK');
-			} else {
-				next();
```

Comment:
Why remove this? You're leaving the request hanging.

Reply:
Shouldn't that ultimately be handled by user with custom 404 hooks?

Reply:
I don't think these will run if you don't call `next`. The request isn't finished here, so the middleware chain needs to continue.